api/v1/controllers/samples.js

```diff
@@ -128,6 +144,7 @@ module.exports = {
    */
   putSample(req, res, next) {
     utils.noReadOnlyFieldsInReq(req, helper);
+    rejectIfNameInBody(req.body);
     doPut(req, res, next, helper);
```

Comment:
what resuing in noReadOnlyFieldsinReq ? The "noReadOnlyFieldsinReq" was created for to check for any read-only fields in requests.